spec/lib/whitehall_importer/create_revision_spec.rb

```diff
@@ -30,22 +30,59 @@
       expect(revision.base_path).to eq("/a-path")
     end
 
-    it "sets backdated_to when edition has been backdated" do
-      backdated_to = Time.zone.now.yesterday.rfc3339
-      whitehall_edition = build(:whitehall_export_edition,
-                                first_published_at: backdated_to)
+    context "when an edition has been backdated" do
```

Comment:
We might need to think about how we phrase these tests.  For example, this currently reads as "when an edition has been backdated it sets backdated_to when the first edition has been backdated" which doesn't make a lot of sense.  Perhaps the contexts could be "when an edition is a first edition" and "when an edition is not a first edition"?  Then the tests could be something like "sets backdated_to when it has been backdated" and "does not set backdated_to when it has not been backdated"?

Reply:
Good point, they didn't make much sense! I've removed the context's all together because as I was rewording the tests it became apparent that they were not adding much, if anything. Hopefully the tests are now clearer!